packages/studio-web/src/app/privacy-dialog.html

```diff
@@ -1,28 +1,33 @@
-  server for preprocessing. The text is not saved and is not used for any other
-  purpose. Using this site means that you agree for your data to be used in this
-  way.
-</div>
-<div style="max-height: none" i18n="Text for analytics" mat-dialog-content>
-  Almost every website you visit will collect data about you using Web Analytics
-  tools which can come with a variety of privacy concerns. We have opted to use
-  <a target="_blank" href="https://plausible.io">Plausible Analytics</a>, which
-  incurs a cost for us but ensures that the data gathered about your visit here
-  is not sold and respects your privacy. We use this data to determine how many
-  people access the site, which devices they use, and other metrics used to make
-  improvements to the site. For a full list of what is collected, please view
-  <a target="_blank" href="https://plausible.io/data-policy"
-    >Plausible's data policy</a
-  >. For a less jargon-y example of the kind of data we see, you can visit this
-  <a target="_blank" href="https://plausible.io/plausible.io"
-    >demonstration site here</a
-  >. In any case, you can opt out entirely by clicking the "Opt Out" button
-  below (you can always opt back in later).
+<div style="max-height: none" mat-dialog-content>
+  <p i18n="Text for privacy dialog">
+    We have built this tool with the top goal of respecting your privacy and
+    data sovereignty. The audio that you use on this site
+    <b>does not get uploaded anywhere</b>. It will stay on your computer. The
+    text you use for this Read Along will be uploaded over an encrypted
+    connection to a server for preprocessing. The text is not saved and is not
+    used for any other purpose. Using this site means that you agree for your
+    data to be used in this way.
+  </p>
+  <p i18n="Text for analytics">
+    Almost every website you visit will collect data about you using Web
+    Analytics tools which can come with a variety of privacy concerns. We have
+    opted to use
+    <a target="_blank" href="https://plausible.io">Plausible Analytics</a>,
+    which incurs a cost for us but ensures that the data gathered about your
+    visit here is not sold and respects your privacy. We use this data to
+    determine how many people access the site, which devices they use, and other
+    metrics used to make improvements to the site. For a full list of what is
+    collected, please view
+    <a target="_blank" href="https://plausible.io/data-policy"
+      >Plausible's data policy</a
+    >. For a less jargon-y example of the kind of data we see, you can visit
+    this
+    <a target="_blank" href="https://plausible.io/plausible.io"
+      >demonstration site here</a
+    >. In any case, you can opt out entirely by clicking the "Opt Out" button
+    below (you can always opt back in later).
+  </p>
 </div>
 
 <div mat-dialog-actions style="display: flex; justify-content: space-evenly">
```

Comment:
Should we add `margin-top: 1em;` to add more space between the text and buttons?
![image](https://github.com/user-attachments/assets/73cb3b1a-ab41-4431-837a-9fc4c916d1dd)

Reply:
Good idea, I agree.

Reply:
Although I'm not sure which line that would belong on...

Reply:
And, fixed.